docs/docs-beta/docs/guides/automation/asset-sensors.md

```diff
@@ -1,10 +1,80 @@
 ---
-title: Asset Sensors
-sidebar_position: 50
+title: Scheduling pipelines across projects and jobs with Asset Sensors
+sidebar_label: Scheduling pipelines across projects and jobs
+sidebar_position: 30
 ---
 
-### Basic Asset Sensor Example
+Asset sensors in Dagster provide a powerful mechanism for monitoring asset materializations and triggering downstream computations or notifications based on those events.
+
+There are many use cases for asset sensors, however the most common is to trigger a job when an asset is materialized in a different job or code location, or to provide custom logic to determine when to trigger a run following an asset materialization.
+
```

Comment:
So far this page doesn't follow our guide structure - is this intentional because it's an MVP?

- What am I going to get out of this guide?
- What do I need to know before diving in, if anything?

Reply:
Ah -- I had meant to add that in. Will update.